### ISSUE TITLE
Aether Issues

### ISSUE THREAD

**Issue by SirStig** [Johni0702/BetterPortals#593]
So I've found two ways to fix this issue.

Either you can downgrade to legacy version 1.4.4 no higher. That will fix the issue and allow you to use better portals with the mod.

If you want the newest version of the Aether, as well as better portals here is another fix.

This will disable better portals for the Aether portal only.

Travel to your config folder where your minecraft/modpack is installed. Find betterportals.cfg Open it with a text editor of any type. Scroll down to The Aether portal. Change Enabled from True to false. Launch the game with the latest update of the Aether and it all runs fine.

This fix allows you to have better portals for all Vanilla portals, twilight forest, and mekanism with the loss of having that nice see through portal for the Aether.

**Comment by coleperrine** [comment]
I would hope this still means it's planned to be fixed, right? Would be insanely dope to use it with Aether down the line.

**Comment by Johni0702** [comment]
> I would hope this still means it's planned to be fixed, right?

No, as mentioned in other issues, I have no intention of spending any more time on a Minecraft version which is almost four years old by now. If you want to use them together, you have to either downgrade Aether or disable BP's Aether support as explained above (that's why I pinned this thread), or find someone able and willing to fix it.

**Comment by coleperrine** [comment]
Understood, thanks for the clarification!

**Comment by minecraftmojang2** [comment]
thank you for telling me. i installed aether 1.4.4. thanks!

**Comment by Blunder-G** [comment]
This is the problem and this is the solution. Thanks man :)

**Comment by LoneWolfGmer** [comment]
it's a bummer the dev won't fix it but not much you can do. if the source code was released someone else could fix it but who knows XD
